src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1173,8 +1173,9 @@ void Ekf2::run()
 			ev_data.quat = q;
 
 			// position measurement error from parameters. TODO : use covariances from topic
-			ev_data.posErr = fmaxf(_ev_pos_noise.get(), fmaxf(ev_pos.eph, ev_pos.epv));
+			ev_data.posErr = fmaxf(_ev_pos_noise.get(), ev_pos.eph);
 			ev_data.angErr = _ev_ang_noise.get();
+			ev_data.hgtErr = fmaxf(_ev_pos_noise.get(), ev_pos.epv);
```

Comment:
The only change required is this one. The changes on the `mavlink_receiver` are going to be replaced by https://github.com/PX4/Firmware/pull/9895.

Reply:
Yeah that's fine, I'm not worried if they get overwritten. I was just thinking it's just a an intermediate step until the ODOM one you are working on is ready :)